tests/test_classifier/test_boundaries.py

```diff
@@ -0,0 +1,337 @@
+# tests.test_boundaries
+# Ensure that the decision boundary visualizations work.
+#
+# Author:   Author:   Nathan Danielsen <nathan.danielsen@gmail.com>
+# Created:  Sun Mar 19 13:01:29 2017 -0400
+#
+# Copyright (C) 2017 District Data Labs
+# For license information, see LICENSE.txt
+#
+# ID: test_knn.py [] nathan.danielsen@gmail.com $
```

Comment:
test_boundaries.py || test_knn.py? 

A note on these ID tags, I have a script that detects them and fills in the brackets with the commit to master that included the original change. This is part of a workflow for a previous place I worked at, but eventually we may want to study our revisions and user growth.